onnxruntime/test/contrib_ops/greedy_search_test.cc

```diff
@@ -50,11 +50,16 @@ TEST(GreedySearchTest, GptGreedySearchFp16_VocabPadded) {
   const char* input_names[] = {"input_ids", "max_length", "min_length", "repetition_penalty"};
   const char* const output_names[] = {"sequences"};
 
+#ifdef USE_CUDA
   constexpr int min_cuda_architecture = 530;
-  if (HasCudaEnvironment(min_cuda_architecture)) {
+  if (!HasCudaEnvironment(min_cuda_architecture)) return;
+#endif
+  {
     Ort::SessionOptions session_options;
 #ifdef USE_CUDA
     Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_CUDA(session_options, 0));
+#elif USE_ROCM
```

Comment:
should be consistent here, either `#if #elif` or `#if defined   #elif defined`

Reply:
Fixed by 668632188427d8173fa3fccb49ef2b98c2634e59.